src/handler/fadingMessageHandler.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+// Models
+let FadingMessage = require("../storage/model/FadingMessage");
+
+// Utils
+let log = require("../utils/logger");
+
+let isLooping = false;
```

Comment:
isn't a global state kinda hacky? I mean, it's a shitposting bot so idc. Just wondering

Reply:
nah man